src/NetworkUtil.php

```diff
@@ -0,0 +1,345 @@
+<?php
+
+namespace Castor;
+
+class NetworkUtil
+{
+    /**
+     * Check if a port is accessible for a host.
+     *
+     * @param int         $port    The port to check
+     * @param string      $host    The host to check (default to 127.0.0.1)
+     * @param int         $timeout The timeout in seconds
+     * @param string|null $name    The name of the port (will be displayed in the output when $quiet is false)
+     * @param bool        $throw   If true, will throw an exception if the port is not accessible instead of returning
+     * @param bool        $quiet   If true, will not output anything
+     *
+     * @return bool True if the port is accessible, false otherwise
+     */
+    public static function wait_for_port(
+        int $port,
+        string $host = '127.0.0.1',
+        int $timeout = 30,
+        string $name = null,
+        bool $throw = false,
+        bool $quiet = false
+    ): bool {
+        $msg = $name ? "Waiting for {$name} ({$host}:{$port}) to be accessible..." : "Waiting for {$host}:{$port} to be accessible...";
+        false === $quiet && io()->write($msg);
+        $successMessage = $name ? " <fg=green> OK {$name} ({$host}:{$port}) is accessible !</>" : " <fg=green> OK {$host}:{$port} is accessible !</>";
```

Comment:
This is the seconde time where you use `$name ? : ` ternary. 
Using a `if ($name) { } else { }` will make the code more readable. Could you change the code?